NEWS.md

```diff
@@ -27,6 +28,141 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+This release is shaping up to be the second in a row that is larger than
+usual both in terms of new features and fixes. As usual, most of the
+commits are by the core devs: @Consolatis, @ahesford, @jlindgren90 and
+@johanmalm, but there are also a few new strong contributors as noted in
+the log.
+
+### Added
+
+- Support libinput config option for calibration matrices.
+  `<libinput><device><calibrationMatrix>`. Written-by: @SnowNF
+- Add new window-switcher field content types `workspace`, `state`,
+  `type_short` and `output`. Written-by: @droc12345 PR #1623
+
+```xml
+<windowSwitcher allWorkspaces="yes">
+  <fields>
+    <field content="workspace" width="5%" />
+    <field content="state" width="3%" />
+    <field content="type_short" width="3%" />
+    <field content="output" width="9%" />
+    <field content="identifier" width="30%" />
+    <field content="title" width="50%" />
+  </fields>
+</windowSwitcher>
+```
+
+- Support input methods (or input method editors, commonly abbreviated
+  IMEs) like Fcitx5 and IBus, using protocols text-input-v3 and
+  input-method-v2. This includes IME popups. Written-by: @tokyo4j
```

Comment:
I think IBus should not mentioned here because IBus does not support input-method-v2 currently so it only uses dbus interface, which isn't affected by changes I made.

Reply:
Have removed IBus. Thanks.